cores/piduino/Arduino.h

```diff
@@ -142,10 +142,17 @@ void loop(void);
 
 void uart_check_fifos();
 
+
+
 #ifdef __cplusplus
 } // extern "C"
 #endif
 
+// WMath prototypes
+long random(long);
+long random(long, long);
+void randomSeed(unsigned long);
+
```

Comment:
those random definitions need to go where makeWord is defined below. That will ensure that the overload is defined only in C++ and C files including the Arduino header will compile fine